packages/data/lib/src/repositories/app_version_repsitory_impl.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:domain/domain.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+const _versionCollection = 'StarMovie_versions';
+
+class AppVersionRepositoryImpl implements AppVersionRepository {
```

Comment:
seems that this is not the best naming. now u have a repo, which solves only one task - check app version. it is more similar to usecase. repo should be named smth like RemoteDBRepo (or some better naming)

Reply:
dat naming.... I'v got Vietnamian flashbacks =) 
(in epam lab mentor liked to write similar  comments)
May be AppInfoRepository will be better in this case instead of RemoteDb, cuz we get here information about app, and may be exists cases to store more app info in cloud...